buildSrc/src/main/kotlin/io/getstream/chat/android/command/utils/Modules.kt

```diff
@@ -0,0 +1,19 @@
+package io.getstream.chat.android.command.dag
+
+import com.squareup.moshi.Types
+import io.getstream.chat.android.command.utils.moshi
+import java.io.File
+
+fun parseModules(modulesFile: File): List<String> {
+    val json = modulesFile.readLines().joinToString()
+    return moshi()
+        .adapter<List<String>>(Types.newParameterizedType(List::class.java, String::class.java))
```

Comment:
```suggestion
        .adapter<List<String>>()
```

This works on its own if you `import com.squareup.moshi.adapter` and add `@OptIn(ExperimentalStdlibApi::class)`